cirq-core/cirq/circuits/circuit_operation_test.py

```diff
@@ -379,11 +389,13 @@ def test_string_format():
     )
     assert (
         str(op2)
-        == f"""\
-[ 0: ───X───X─── ]
-[           │    ]
-[ 1: ───H───@─── ](qubit_map={{1: 2}}, parent_path=('outer', 'inner'),\
- repetition_ids=['a', 'b', 'c'])"""
+        == """\
+┌────────────────────────────────────────────────────────────────────────────────┐
+│0: ───X───X───                                                                  │
+│          │                                                                     │
+│1: ───H───@───                                                                  │
+│qubit_map={1: 2}, parent_path=('outer', 'inner'), repetition_ids=['a', 'b', 'c']│
```

Comment:
I'd prefer to keep these arguments outside the box, especially since this is related to #3612. The circuit is the "reusable" portion, while the arguments may change from instance to instance.